desk/app/channels.hoon

```diff
@@ -2602,6 +2613,10 @@
       ?+  -.inline  |
         ?(%bold %italics %strike %blockquote)  ^$(p.verse p.inline)
         ?(%code %inline-code)                  $(inline p.inline)
+      ::
+          %sect
+        ?~  p.inline  $(inline '@all@everyone@channel')
+        $(inline p.inline)
```

Comment:
For consistency, shouldn't specific sects also get an `@` prefix?

The whole thing is slightly questionable anyway, since it ties you to client-side input/representation logic, but I don't think there's a better answer if you want to include these in search intuitively. But, `@all@everyone@channel` should probably be checked as separate strings.

```suggestion
          %sect
        ?.  =(~ p.inline)
          $(inline (cat 3 '@' p.inline))
        ?|  $(inline '@all')
            $(inline '@everyone')
            $(inline '@channel')
        ==
```

Reply:
Also am I blind or do we just ignore `%ship` nodes lol

Reply:
If we don't want to couple search with client representation the solution would be to perform structured search. This would require client to submit a search `inline`/`verse` rather than a string as it currently does, but it does not seem overly difficult.

Reply:
yeah we should look into that. I will just use `@all` for now and make sure sects are prefixed with `@` as well